cves/CVE-2010-1157.yml

```diff
@@ -44,36 +44,33 @@ description_instructions: |
   expressions" is fine, and security phrases like "invalid write" are fine to
   keep too.
 description: |
-  A portion of Tomcat that handles http authentication via DIGEST and BASIC strategies had the potential to allow
-  for the viewing of the local name or IP address of the server running Tomcat. The header for the authentication 
-  via these strategies includes a <realm-name> element.
-  If a <realm-name> element is specified for the application in one of the configuration files, it will be used. 
-  However, if a <realm-name> is not specified then Tomcat will generate a realm name and
-  in some circumstances this can expose the local host name or IP address of the machine running Tomcat. 
+  May allow remote attackers to discover the server's hostname or IP address by 
+  sending a request for a resource that requires one of two specific 
+  authentication levels.
 
 bounty_instructions: |
   If you came across any indications that a bounty was paid out for this
   vulnerability, fill it out here. Or correct it if the information already here
   was wrong. Otherwise, leave it blank.
 bounty:
   amt:
-  announced: 2010-04-23
+  announced:
   url:
 bugs: []
 fixes_vcc_instructions: |
   Please put the Git commit SHA in "commit" below, and any notes about how this
   was discovered in the "note" field.
   Refer to our instructions on how to find a Git SHA from an SVN revision.
 fixes:
-   - commit: 78e45a225c5c5564945191f7e421f8859885d63b
-     note: SVN rev 936541, from the Tomcat website.
-   - commit:
-     note:
+   - commit: a4de93b2aaf74fff7c1e194f7ce3d2228a9680b8
+     note: |
+      SVN rev 936539, from the Tomcat website. The fix was to add a variable for
+      the default behavior of the realm.
 vccs:
-  - commit: 2ee172f418591edd61cfd3dc9676753b281d1fe2
-    note: The git history for this directory only goes back to a branching of Tomcat 5.5.x, and does not reveal when original implementation was made
-  - commit:
-    note:
+  - commit: c4762411946378b1bdf758f16b15845f9646f9d5
+    note: |
+      The original VCC was part of modifying realm was recieved from the request
+      which resulted in the vulnerability.
```

Comment:
I was not able to completely understand what you were trying to say here. Changing this sentence to something a bit more structured would make it easier to read, specifically, the "The original VCC was part of modifying realm was received" portion of it.

Reply:
Updated this sentence for clarity. Also reviewed the commit and determined that this was not the proper VCC.